src/components/FeaturedRepositories/FeaturedRepositories.tsx

```diff
@@ -1,95 +1,108 @@
 
 const maxWidth = 80;
-const projects = [
+const projects : Array<OrgRepo> = [
   {
-    title: 'Ethereum/EIPs',
-    description: 'The Ethereum Improvement Proposal repository',
-    imageUrl: 'https://avatars1.githubusercontent.com/u/6250754',
-    link: '/ethereum/EIPs',
+    org: 'ethereum',
+    repo: 'EIPs'
   },
   {
-    title: 'Compound Protocol',
-    description: 'The Compound On-Chain Protocol',
-    imageUrl: 'https://avatars1.githubusercontent.com/u/32911405',
-    link: '/compound-finance/compound-protocol',
+    org: 'compound-finance',
+    repo: 'compound-protocol'
   },
   {
-    title: 'Gnosis Safe Multisig',
-    description: 'The most trusted platform to store digital assets on Ethereum',
-    imageUrl: 'https://avatars1.githubusercontent.com/u/24954468',
-    link: '/gnosis/safe-react',
+    org: 'gnosis',
+    repo: 'safe-react'
   },
   {
-    title: 'Pocket Core',
-    description: 'Official implementation of the Pocket Network Protocol',
-    imageUrl: 'https://avatars0.githubusercontent.com/u/33689860',
-    link: '/pokt-network/pocket-core',
+    org: 'pokt-network',
+    repo: 'pocket-core'
   },
   {
-    title: 'yearn-finance',
-    description: 'DeFi made simple',
-    imageUrl: 'https://avatars1.githubusercontent.com/u/60162948',
-    link: '/iearn-finance/yearn-protocol',
+    org: 'iearn-finance',
+    repo: 'yearn-protocol'
   },
   {
-    title: 'tBTC',
-    description: 'Trustlessly tokenized Bitcoin on Ethereum',
-    imageUrl: 'https://avatars0.githubusercontent.com/u/30537626',
-    link: '/keep-network/tbtc',
+    org: 'keep-network',
+    repo: 'tbtc'
   },
   {
-    title: 'The Commons Stack',
-    description: 'Realigning Incentives to Support Public Goods',
-    imageUrl: 'https://avatars2.githubusercontent.com/u/48513475',
-    link: '/commons-stack/iteration0',
+    org: 'commons-stack',
+    repo: 'iteration0'
   },
   {
-    title: 'Tokenlog',
-    description: 'Token-weighted backlogs',
-    imageUrl: 'https://tokenlog.xyz/icon.png',
-    link: '/wslyvh/tokenlog',
+    org: 'wslyvh',
+    repo: 'tokenlog'
   },
   {
-    title: 'HausDAO',
-    description: 'Community governance of the DAOhaus ecosystem',
-    imageUrl: 'https://avatars3.githubusercontent.com/u/69052185',
-    link: '/HausDAO/pokemol-web',
+    org: 'HausDAO',
+    repo: 'pokemol-web'
   },
 ];
 
+const repoData = {
+  loading: true,
+  repos: new Array<Repository>(),
+};
+
+
 export function FeaturedRepositories() {
+  const [data, setData] = useState(repoData);
+  const repos = Array<Repository>();
+
+  useEffect(() => {
+    async function asyncEffect() {
+      const featuredRepos = [...await RepositoryService.GetRepositories(), ...projects].filter(
+        (repo, i, arr) => arr.findIndex(t => t.repo === repo.repo && t.org === repo.org) === i
+      );
+      for (let i = 0; i < featuredRepos.length; i++) {
+        repos.push(await IssueService.GetRepository(featuredRepos[i].org, featuredRepos[i].repo));
```

Comment:
With the current design this is probably the easiest way to fetch repository with votes + extend them with more details (image, description, etc.). I wonder though if this won't be too much of a performance hit. Especially considering this component is used on the index. 

Have to think about the design and data model. We could extend the repository model to include more metadata. Alternatively, I'd be ok with not having real-time data. We could create a pre-build script that fetches all the required data and creates a static list/saves a json file in the solution (much like the pre-defined projects list).

Reply:
For now in #34 I have added a 'Show more' button, it will load 5 repos at a time.

If it's alright to have not real-time data I would agree to store it in the DB, but when do you update it? When someone makes a vote?

Reply:
I was actually thinking to not even hit the database. Get the repo's pre-build, save it in a json response within the solution and serve those. The data might get stale quickly, but will be updated once we trigger a new build. Could add a nightly build or something so repos are at least updated daily.